src/components/Page/Base.js

```diff
@@ -1,19 +1,25 @@
+/* eslint-disable no-console */
+/* eslint-disable no-shadow */
 import { NextSeo } from "next-seo";
 import PropTypes from "prop-types";
 import React from "react";
 
 import Footer from "@/pesayetu/components/Footer";
 import { footerArgs } from "@/pesayetu/config";
+import getFooterMenu from "@/pesayetu/functions/menus/getFooterMenu";
 /**
  * Base page that can be used to build all other pages.
  */
-
-function BasePage({ children, ...props }) {
+function BasePage({ children, menus, ...props }) {
+  const footerProps = {
+    ...footerArgs,
+    footerArgs: getFooterMenu(menus?.footerMenu ?? []),
```

Comment:
My understanding is all data from CMS will be loaded in `getStaticProps` of a given page. Has this changed?